front/src/modules/ui/components/buttons/Button.tsx

```diff
@@ -3,26 +3,30 @@ import styled from '@emotion/styled';
 
 import { rgba } from '@/ui/themes/colors';
 
-type Variant =
+export type ButtonVariant =
   | 'primary'
   | 'secondary'
   | 'tertiary'
   | 'tertiaryBold'
   | 'tertiaryLight'
   | 'danger';
 
-type Size = 'medium' | 'small';
+export type ButtonSize = 'medium' | 'small';
+
+export type ButtonPosition = 'left' | 'middle' | 'right' | 'solo';
```

Comment:
@FelixMalfait Instead of putting `solo`, I rather just check if position is not `undefined`, if this one is `undefined` the `Button` is not in a group